clock_exercise/AlarmSettingPage.qml

```diff
@@ -48,11 +60,13 @@ Rectangle {
     AlarmSlider {
         id: alarmSlider
 
-        anchors.horizontalCenter: parent.horizontalCenter; y:575
+        anchors.horizontalCenter: parent.horizontalCenter
+        y: 575
     }
 
     BigButton {
-        x: 23; y: 706
+        x: 23
+        y: 706
```

Comment:
Non so se tu abbia attivato il formattatore di QML oppure se queste modifiche tu le abbia fatte a mano, in ogni caso solitamente per questo genere di cambiamenti si fanno dei commit appositi con una descrizione del tipo "formattazione QML", così chi vede i commit sa che lì ci sono solo cambiamenti di formattazione, e che non serve dedicarci del tempo per fare review, e si possono praticamente skippare.

Strumenti come `git cola` in questo caso sono molto utili se per puro caso al salvataggio il formattatore è partito su un file appena modificato e ti ha cambiato cose a giro che non ti riguardano.